pkg/ebpf/tracee.go

```diff
@@ -312,7 +314,10 @@ func New(cfg Config) (*Tracee, error) {
 			return t, fmt.Errorf("could not get event")
 		}
 		for _, capArray := range evt.Dependencies.Capabilities {
-			caps.Require(capArray)
+			err = caps.Require(capArray)
+			if err != nil {
+				logger.Debug("Requiring capabilities", "error", err)
```

Comment:
This should be error or even fatal, as it will prevent tracee from behaving as required by the user.

Reply:
I'm changing them to Fatal but I'm not aborting the logic.